packages/fx-core/src/component/coordinator/lifecycleStage.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { DriverContext, Step } from "../interface/buildAndDeployArgs";
+import { PrerequisiteError } from "../error/componentError";
+import { StepDriver } from "../interface/stepDriver";
+import { Container } from "typedi";
+
+/**
+ * entrance of the all lifecycle stage
+ */
+class LifecycleStage {
+  static async runSteps(steps: Step[], context: DriverContext) {
+    const drivers = steps.map((it) => Container.get(it.driver) as StepDriver);
+    if (drivers.find((it) => !it)) {
```

Comment:
I don't understand the logic of this part of code? Can you explain more?

Reply:
What the error handling logic when there is some driver name invalid? Will it break the whole execution?

Reply:
Yea, It's just some sample codes without error handling.